src/app/(spaces)/s/[handle]/ProfileSpace.tsx

```
@@ -23,9 +23,13 @@ export const ProfileSpace = ({
     return <SpaceNotFound />;
   }
 
-  const INITIAL_PERSONAL_SPACE_CONFIG = createIntialPersonSpaceConfigForFid(
-    spaceOwnerFid,
-    spaceOwnerUsername ?? undefined,
+  const INITIAL_PERSONAL_SPACE_CONFIG = useMemo(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)